mapbox_maps_flutter_mobile/lib/src/mapbox_map.dart

```diff
@@ -458,6 +464,7 @@ class MapboxMap extends ChangeNotifier {
           options);
 
   /// Queries the map for rendered features with one typed featureset.
+  @experimental
```

Comment:
Is tagging them experimental intentional?

Reply:
nope, I have no idea why it got here, will remove